docs/src/content/docs/es/getting-started.mdx

```diff
@@ -49,11 +51,24 @@ Prueba Starlight en tu navegador:
 
 ## Creando contenido con Starlight
 
-Starlight admite la creación de contenido en Markdown y MDX.
+Starlight está listo para que añadas nuevo contenido o traigas tus archivos existentes.
+
+### Formatos de archivo
+
+Starlight admite la creación de contenido en Markdown y MDX. (Puedes agregar compatibilidad con Markdoc instalando la integración experimental de [Astro Markdoc](https://docs.astro.build/en/guides/integrations-guide/markdoc/).)
```

Comment:
The Markdoc page isn’t translated, but probably still makes sense to point to the Spanish version so that readers are in the Spanish site:

```suggestion
Starlight admite la creación de contenido en Markdown y MDX. (Puedes agregar compatibilidad con Markdoc instalando la integración experimental de [Astro Markdoc](https://docs.astro.build/es/guides/integrations-guide/markdoc/).)
```